core/sail/lmdb/src/main/java/org/eclipse/rdf4j/sail/lmdb/ValueStore.java

```diff
@@ -198,7 +200,7 @@ public static void main(String[] args) throws Exception {
 				String ns = valueStore.data2namespace(data);
 				System.out.println("[" + id + "] " + ns);
 			} else {
-				Value value = valueStore.data2value(id, data);
+				Value value = valueStore.data2value(id, data, null);
 				System.out.println("[" + id + "] " + value.toString());
```

Comment:
Just noticed a couple of `System.out.println` statements here - I assume those are there for debugging/testing purposes? No problem to, just don't forget to take them out when we're done / ready to release.

Reply:
This was a copy of existing code from NativeStore. I've just removed the main method from ValueStore.